libpam/pam_get_authtok.c

```diff
@@ -174,6 +175,14 @@ pam_get_authtok_internal (pam_handle_t *pamh, int item,
       (chpass > 1 && resp[1] == NULL))
     {
       /* We want to abort */
+      if (resp[0]) {
+        pam_overwrite_string (resp[0]);
+        _pam_drop (resp[0]);
+      }
+      if (resp[1]) {
+        pam_overwrite_string (resp[1]);
+        _pam_drop (resp[1]);
+      }
```

Comment:
Since both `pam_overwrite_string` and `_pam_drop` check their arguments, let's not duplicate the check here.